### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by edgan** [hexchat/hexchat#2519]
There hasn't been a new release since Dec 21, 2019, coming up on a year ago. While using Fedora 32 I downgraded my version of pango to workaround this underscore issue. It has since been fixed in master, and I have upgraded to Fedora 33. The downgrade of pango on Fedora 33 causes nautilus to crash with "nautilus: symbol lookup error: nautilus: undefined symbol: pango_attr_insert_hyphens_new". I tried taking the PR patches and merging them into the official 2.14.3 release. What I found is neither applies cleanly, because so much has changed since the last release. I could guess as to the correct way to merge them, but I found it easier to just rebuild the src.rpm with a copy of master. This solves the problem for me, but not for anyone else using a package based on the official release.

Why hasn't there been a release? When can we expect one?

https://github.com/hexchat/hexchat/issues/2449
https://github.com/hexchat/hexchat/pull/2500

**Comment by v00v** [comment]
Yeah, sadly HexChat development are very slow. But I would rather see more bug fixes than major feature implementation :-) Still I think that @TingPing does a very good job, considering low resources. Maybe everyone could donate more to support him, than he would do more releases ^_^

**Comment by TingPing** [comment]
The python plugin needs a few fixes then I'd do a release.

**Comment by FunctionNotFound** [comment]
@TingPing did not see this post, before I posted https://github.com/hexchat/hexchat/issues/2548 sorry.
